src/SoccerStatistics.Api.Application/Handlers/GetAllTeamsHandler.cs

```diff
@@ -17,7 +17,7 @@ public GetAllTeamsHandler(ITeamService service)
             _service = service;
         }
 
-        public async Task<IEnumerable<TeamBasicDTO>> Handle(GetAllTeamsQuery request, CancellationToken cancellationToken) 
+        public async Task<IEnumerable<TeamDTO>> Handle(GetAllTeamsQuery request, CancellationToken cancellationToken) 
             => await _service.GetAllAsync();
     }
```

Comment:
Why switch it to more complex dto?

Reply:
While getting all leagues and stadiums we have complex dto too. That's the only reason why I changed it.

Reply:
I can revert changes to previous state :p But we should talk about this at the next meeting with our UI/UX designer (what specific information we store in list of teams in our apps)